cdap-docs/_common/common-build.sh

```diff
@@ -131,8 +131,11 @@ function usage() {
 
 function clean() {
   cd $SCRIPT_PATH
-  rm -rf $SCRIPT_PATH/$BUILD
-  mkdir $SCRIPT_PATH/$BUILD
+  if [ -d "${SCRIPT_PATH}/${BUILD}" ]
```

Comment:
This existence check is redundant with the rm ``-f`` flag.  The ``-f`` flag causes it to return 0 if it doesn't exist.